lib/js_locale_helper.rb

```diff
@@ -221,11 +221,14 @@ def self.output_extra_locales(bundle, locale)
 
     return "" if translations.blank?
 
-    output = +"if (!I18n.extras) { I18n.extras = {}; }"
-    locales.each { |l| output << <<~JS }
+    # output = +"if (!I18n.extras) { I18n.extras = {}; }"
+    locales.each do |l|
+      translations_json = translations[l].to_json
+      output << <<~JS
         if (!I18n.extras["#{l}"]) { I18n.extras["#{l}"] = {}; }
-        Object.assign(I18n.extras["#{l}"], #{translations[l].to_json});
+        Object.assign(I18n.extras["#{l}"], #{translations_json});
```

Comment:
(refactored this slightly because the syntax was breaking syntax highlighting in vscode 🤷‍♂️)